cli/src/feature.rs

```diff
@@ -344,7 +344,7 @@ fn feature_activation_allowed(rpc_client: &RpcClient, quiet: bool) -> Result<boo
         )
         .unwrap_or((false, false));
 
-    if !stake_allowed && !rpc_allowed && !quiet {
+    if !(quiet || stake_allowed && rpc_allowed) {
```

Comment:
```suggestion
    if !quiet && !(stake_allowed && rpc_allowed) {
```
Wdyt? I think it would be helpful to see the feature-version info on all `!quiet` calls.